test/test_muscles.cpp

```diff
@@ -291,8 +241,8 @@ TEST(hillType, copy)
         insertion.setName(newName);
         hillType.updateOrientations(model, Q, qDot, 2);
 
-        EXPECT_NEAR(hillType.position().length(), 0.066381977535807504, requiredPrecision);
-        EXPECT_NEAR(shallowCopy.position().length(), 0.066381977535807504, requiredPrecision);
+        EXPECT_NEAR(hillType.position().length(), 0.07570761027741163, requiredPrecision);
+        EXPECT_NEAR(shallowCopy.position().length(), 0.07570761027741163, requiredPrecision);
```

Comment:
Is there a reason why these values changed?

Reply:
The values are supposed to change because I modify the geometry. Last time I ran the program, it worked with the original values and I did not understand why. Now it is working properly.